src/main/java/io/quarkus/bot/PingWhenNeedsTriageRemoved.java

```diff
@@ -33,7 +34,7 @@ void pingWhenNeedsTriageRemoved(@Issue.Unlabeled GHEventPayload.Issue issuePaylo
         GHIssue issue = issuePayload.getIssue();
         GHLabel removedLabel = issuePayload.getLabel();
 
-        if (!removedLabel.toString().equals(Labels.TRIAGE_NEEDS_TRIAGE)) {
+        if (!Labels.TRIAGE_NEEDS_TRIAGE.equals(removedLabel.getName())) {
```

Comment:
2 important things here:
* when you have a constant, always use it first to compare: it completely avoid NPE as you are sure your constant is not null. In this case, it's not very important but it's good practice to always do it this way
* don't use the `toString()` of `GHLabel`. Most of the time `toString()` is technical. It could be `GHLabel[name=myname, color=mycolor]` for instance. You need to compare to the proper field (here `getName()`).

Reply:
Yes, I was just debugging it and got the knowledge that if we will not use `getName()` then will end up getting something like this - 
```
org.kohsuke.github.GHLabel@71997c84
```

![image](https://user-images.githubusercontent.com/46818757/114276153-1920d300-9a43-11eb-8cb5-57058c97a839.png)